### ISSUE TITLE
 Error: Cannot find module 'ethereumjs-wallet/hdkey'

### ISSUE THREAD

**Issue by marekkirejczyk** [trufflesuite/ganache-core#137]
After `ethereumjs-wallet` 0.6.1 version release, I started getting: `Cannot find module 'ethereumjs-wallet/hdkey`, when trying to require/import `ganache-core`.

I guess fixing version in `package.json` would be an easy fix, so use `"ethereumjs-wallet": "0.6.0"`, instead of `"ethereumjs-wallet": "~0.6.0"`.

Full error:
```
     Error: Cannot find module 'ethereumjs-wallet/hdkey'
      at require (internal/module.js:11:18)
      at Object.<anonymous> (node_modules/ganache-core/lib/statemanager.js:11:13)
      at Object.require.extensions.(anonymous function) [as .js] (node_modules/babel-register/lib/node.js:152:7)
      at require (internal/module.js:11:18)
      at Object.<anonymous> (node_modules/ganache-core/lib/subproviders/geth_api_double.js:3:20)
      at Object.require.extensions.(anonymous function) [as .js] (node_modules/babel-register/lib/node.js:152:7)
      at require (internal/module.js:11:18)
      at Object.<anonymous> (node_modules/ganache-core/lib/provider.js:7:21)
      at Object.require.extensions.(anonymous function) [as .js] (node_modules/babel-register/lib/node.js:152:7)
      at require (internal/module.js:11:18)
      at Object.<anonymous> (node_modules/ganache-core/index.js:1:16)
      at Object.require.extensions.(anonymous function) [as .js] (node_modules/babel-register/lib/node.js:152:7)
      at require (internal/module.js:11:18)
      at _callee$ (test/etest.js:18:21)
      at tryCatch (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:62:40)
      at Generator.invoke [as _invoke] (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:296:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/babel-runtime/node_modules/regenerator-runtime/runtime.js:114:21)
      at step (node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
      at /Users/marekkirejczyk/Projects/ambrosus/tmp/tmp/node_modules/babel-runtime/helpers/asyncToGenerator.js:35:14
      at new Promise (<anonymous>)
      at new F (node_modules/core-js/library/modules/_export.js:36:28)
      at Context.<anonymous> (node_modules/babel-runtime/helpers/asyncToGenerator.js:14:12)
```

**Comment by tmashuang** [comment]
https://github.com/ethereumjs/ethereumjs-wallet/issues/64

**Comment by holgerd77** [comment]
This should now have been fixed with the [v0.6.2](https://github.com/ethereumjs/ethereumjs-wallet/releases/tag/v0.6.2) release (confirm would be nice), sorry for the inconvenience.
